src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -126,11 +166,26 @@ export default class StudyTagsTooltip extends React.Component<
                 mouseEnterDelay={this.props.mouseEnterDelay}
                 placement={this.props.placement}
                 overlay={
-                    <StudyInfoOverlay
-                        studyDescription={this.props.studyDescription}
-                        studyId={this.props.studyId}
-                        isVirtualStudy={this.props.isVirtualStudy}
-                    />
+                    // for the lock icon (infoIcon false), if there are no placeholders show directly the global message
```

Comment:
Change:

// for the lock icon tooltip, show the message defined by skin_home_page_unauthorized_studies_global_message property immediately if it does not contain placeholders.